llama.cpp

```diff
@@ -2082,6 +2082,11 @@ int llama_get_kv_cache_token_count(struct llama_context * ctx) {
 
 #define LLAMA_MAX_RNG_STATE 64*1024
 
+void llama_set_rng_seed(struct llama_context * ctx, uint32_t seed)
+{
+    ctx->rng.seed(seed);
+}
```

Comment:
Please move the opening brace to the function declaration line, to follow the same convection as the rest of the code.